pkg/minikube/cluster/cluster.go

```diff
@@ -198,7 +198,7 @@ func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	defer response.Body.Close()
 
 	if response.StatusCode != http.StatusOK {
-		return fmt.Errorf("Received %d response from %s while trying to download minikube.iso", response.StatusCode, m.MinikubeISO)
+		return fmt.Errorf("Received %d response from %s while trying to download minishift.iso", response.StatusCode, m.MinikubeISO)
```

Comment:
So what is it? Orginally this said minikube, so I changed it to minishift. We should specify what it is that we're trying to download.